### DISCUSSION TITLE
Can't override icon by file extension

### DISCUSSION THREAD

**Discussion by ubmit** [LazyVim/LazyVim#2707] [Q&A] [answered]
hey there!

i've been trying to override the icons for `.astro` files, but my config seems to not be changing anything. regardless of what string i use, the icon is always a `*`. here is my current settings: https://github.com/ubmit/nvim/blob/main/lua/plugins/devicons.lua am i missing something? 🤔 

thanks in advance ✨

**Comment by dpetka2001** [comment, marked as answer]
```lua
return {
  "nvim-tree/nvim-web-devicons",
  -- TODO: override icons for astro files
  opts = {
    strict = true,
    override_by_extension = {
      ["astro"] = {
        icon = "",
        color = "#EF8547",
        name = "Astro",
      },
    },
  },
}
```
This seems to work.

**Reply by ubmit** [reply]
thank you! @dpetka2001
